index.html

```diff
@@ -8,22 +8,44 @@
   <title>Awsome Books</title>
 </head>
 <body>
+  <nav>
+    <h2>Awesome Books</h2>
+    <ul>
+      <li id="list">List</li>
```

Comment:
- [ ] [OPTIONAL] Please make sure after making all the new changes your add book function is still working as currently when I try to add book I see following error and no book is showing in the list 

<img width="453" alt="Screenshot 2023-07-05 at 4 46 48 PM" src="https://github.com/agneta1991/awsome-books/assets/44798044/ffb70fde-b585-4d1d-94e7-1013a0ad0c4c">

Reply:
It's extraordinary that I double-checked everything, and it's working fine for me and my partner !!